recipes/drogon/all/conanfile.py

```diff
@@ -70,6 +74,7 @@ def configure(self):
             del self.options.with_mysql
             del self.options.with_sqlite
             del self.options.with_redis
+            del self.options.with_spdlog
```

Comment:
```suggestion
           self.options.rm_safe("with_spdlog")
```

double deletion is an error in Conan 2.x.